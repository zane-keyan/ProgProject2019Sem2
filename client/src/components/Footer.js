import React, { Component } from "react";


class Footer extends Component {
  render() {
    return (
      <footer className="footer">
        <div className="container">
          <span className="text-muted">
            <a href="https://github.com/rmit-s3607212-mohammed-alsharifi/ProgProject2019Sem2">
              {" "}
              Team 0
            </a>{" "}
            &#169; {new Date().getFullYear()} - RMIT Project 1 2019
          </span>
        </div>
      </footer>
    );
  }
}

export default Footer;
