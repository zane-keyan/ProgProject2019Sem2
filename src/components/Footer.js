import React, { Component } from "react";


class Footer extends Component {
  render() {
    return (
      <footer className="footer">
        <div className="container">
          <span className="text-muted">
            <a href="https://github.com/Abdosalah/ProgProject2019Sem2">
              {" "}
              Team 0
            </a>{" "}
            &#169; {new Date().getFullYear()}
          </span>
        </div>
      </footer>
    );
  }
}

export default Footer;
