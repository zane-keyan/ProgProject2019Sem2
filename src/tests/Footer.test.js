import React from "react";
import renderer from "react-test-renderer";
import Footer from "../components/footer";

it("render correctly Footer Component", () => {
  const FooterComponent = renderer.create(<Footer />).toJSON();
  expect(FooterComponent).toMatchSnapshot();
});
