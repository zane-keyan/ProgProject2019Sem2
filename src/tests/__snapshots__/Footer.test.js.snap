// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Footer Component 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <span
      className="text-muted"
    >
      <a
        href="https://github.com/Abdosalah/ProgProject2019Sem2"
      >
         
        Team 0
      </a>
       
      Â© 
      2019
       - RMIT Project 1 2019
    </span>
  </div>
</footer>
`;
